benchmarks/distributed_tree_driver/distributed_tree_driver.cpp

```diff
@@ -252,6 +253,10 @@ int main_(std::vector<std::string> const &args, const MPI_Comm comm)
                                                                               "local clouds form a board, 3 -> local clouds form a box." )
         ( "do-not-perform-knn-search", "skip kNN search" )
         ( "do-not-perform-radius-search", "skip radius search" )
+        ( "shift-queries" , "Shift the the local box queries are created in from the point cloud. "
```

Comment:
I don't understand what the first sentence means. Also there is a typo.

Reply:
What would help you? Simply deleting the first sentence?

Reply:
Sure but don't forget to change the rest of the comment `...shrinks this box...`